planning/backlog.md

```diff
@@ -9,15 +9,114 @@ create your own!
 
 > these are necessary for basic usability
 
-- [ ] As a ... I want to ... so that ...
-  - _Given [context] when [a specific action is performed] then [a set of
-    consequences should occur]_
+### As a User
+
+- I would like to see the organization's [Job advisor] name
+
+  - [ ] _there is a prominent logo at the top of the web page_
+
+  - I would like to easily accessing the page/information of the
+    organization(Job advisor).
+
+- [ ] _there should be a navbar to aid visitors in navigate the page in the
+      website._
+
+- I would like to know more information about the organization (job advisor)
+
+  - [ ] _there is a footer on all pages with contact info for social media and
+        GitHub._
+
+- I would like to know the value and impact that organisation (Job advisor) have
+  on the community/sector.
+
+  - [ ] _there is M & V, and impact section which describes the goals and values
+        of the organization (Job advisor)._
+
+### As a Job seeker
+
+- I would like to know some infomartaion about the job vacanies without
+  registration
+
+  - [ ] _there's is a search bar where user can search using keyword without
+        registration._
+
+- I would like to know some infomartaion about companies which have Gender
+  equality
+
+  - [ ] _there's is a "Gender equality" button below search bar which contains
+        brief information about the vacancies offered for freshers_
```

Comment:
Why only for freshers (candidate without job experience)? Experience job seekers might  also be interested in gender equality?

Reply:
Then let me move it under jobseeker